src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVED_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] The value of your action types should follow the _`ducks pattern`_. Kindly change these definitions to the following code snippet.

  ```
  const ADD_BOOK = 'bookStore/books/ADD_BOOK';
  const REMOVED_BOOK = 'bookStore/books/REMOVE_BOOK';
  ```